tests/new_install.yml

```diff
@@ -0,0 +1,23 @@
+---
+
+- hosts: all
+  vars:
+    docker_version: 1.12.1
+    ob: '{'
```

Comment:
I guess there was some quoting struggle here.  Calling this open_brace would be easier for me to understand.  A comment about the struggle might help too.